src/services/milestones/milestones.hooks.js

```diff
@@ -241,6 +251,14 @@ const storePrevState = () => context => {
   return context;
 };
 
+const convertTokenToTokenAddress = () => context => {
```

Comment:
This should be removed too, like above

Reply:
Yes we remove all entities that already exists in DB with migration, but what about the milestone and donations that will create after the migration?
if we want to don't use this , so we should change front to send `tokenAddress` instead of sending `token`